<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AR600MainWindow</class>
 <widget class="QMainWindow" name="AR600MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,10" columnstretch="0,10,0">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="PowerLayout"/>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="DriverLogWidgetLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
       </layout>
      </item>
      <item row="0" column="2" rowspan="2">
       <layout class="QGridLayout" name="gridLayout_5" rowstretch="2,8">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="CommandControlLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="DriverControlLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QVBoxLayout" name="ChannelTableLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpenConfigFile"/>
    <addaction name="actionSaveConfigFile"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Соединение</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionOpenConnectConfig"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Питание</string>
    </property>
    <addaction name="actionOn"/>
    <addaction name="actionOff"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="MainToolBar">
   <property name="windowTitle">
    <string>Панель настроек</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpenConfigFile">
   <property name="text">
    <string>Открыть файл настроек</string>
   </property>
  </action>
  <action name="actionSaveConfigFile">
   <property name="text">
    <string>Сохранить файл настроек</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Подключиться</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Отключиться</string>
   </property>
  </action>
  <action name="actionOn">
   <property name="text">
    <string>Включить все</string>
   </property>
  </action>
  <action name="actionOff">
   <property name="text">
    <string>Выключить все</string>
   </property>
  </action>
  <action name="actionOpenConnectConfig">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
